.github/workflows/pr.yml

```
@@ -0,0 +1,41 @@
+name: Pull Request Checker
+on:
+  pull_request:
+    types:
+      - opened
+      - reopened
+    branches:
+      - master
+
+jobs:
+  check-pr:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/github-script@v6
+      with:
+        script: |
+          const pr = await github.rest.pulls.get({
+            owner: context.repo.owner,
+            repo: context.repo.repo,
+            pull_number: context.issue.number
+          })
+          const isTitleValid = /^\[#\d\] /.test(pr.data.title)
```

There might be some issues with the regex.
```suggestion
          const isTitleValid = /^\[#\d+\] /.test(pr.data.title)
```

---

Thanks for the catch!